woocommerce/woocommerce
# Add support for connecting a site to WooCommerce.com via CLI

### All Submissions:






### Changes proposed in this Pull Request:



This PR adds the command `wp wc com connect`, which allows connecting a via CLI. 

The command check if the site is already connected and proceed to output a URL to continue authorization.

Closes # .

### How to test the changes in this Pull Request:

1. Checkout this branch.
2. Run `wp wc com connect`
3. If the site is not already connected the command should output a URL to continue authorization.
![Screenshot 2022-07-08 at 18 00 05](https://user-images.githubusercontent.com/1531403/178028608-a364a888-25b2-45da-b8a5-2ee1eade8102.png)
4. User should be able to connect the site to WooCommerce.com by visiting the URL.

### Other information:




### FOR PR REVIEWER ONLY:
